tests/api-testing/ci/Dockerfile.openobserve

```diff
@@ -0,0 +1,8 @@
+# syntax=docker/dockerfile:1
+
+FROM public.ecr.aws/debian/debian:bullseye-slim as runtime
+RUN apt-get update && apt-get install -y --no-install-recommends ca-certificates curl htop iftop sysstat procps lsof net-tools
```

Comment:
Lines #4 and #5 have no use during testing, those tools are only useful if exec into a container to check networking issues